### DISCUSSION TITLE
👩‍🌾 Sunflower Fashion Week #3

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2639] [Art]
Did you know life is about more than just plating potatoes? We also care about fashion! 💅💃

Welcome to the 3rd Bumpkin Competition a.k.a Sunflower Fashion Week!

The Art Team will choose the best outfits and announce the winners on the first week of every month! 😍

There's no specific theme for this competition! Designers are encouraged to use their imagination and come up with original ideas. 

Before continuing, it is recommended to read the Bumpkin Guideline, which can be found here:

https://docs.sunflower-land.com/contributing/for-designers/bumpkin-guidelines

## Requirements ✏️

- Dimensions of file should be 100x100px.
- The layer (e.g. shirt) should be on it's own layer without any other parts. When placed on top of a Bumpkin it should perfectly align with its body.
- Art should be original and not plagiarised.
- If submitting multiple items, an aseprite file would be appreciated to speed up review. 🤗
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely. Players may trade your item without the need to pay you royalties. The community may make changes and improve artwork that is submitted.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/bumpkin-art-competition#terms-and-conditions) of the game.
- Bumpkins live inside of the game! Please include an NPC version of your design that can be used inside of Sunflower Land.

When submitting art, please include the following:

- Your farm ID.
- A minified version of your art piece.
- An example of the art used inside of the game or on the Bumpkin. Here's a nice submission example from a community artist: https://github.com/sunflower-land/sunflower-land/discussions/1748?sort=new#discussioncomment-5106261

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however this competition is not just about the prize money. It's an opportunity to showcase your incredible talent and become an essential part of the Sunflower Land experience. 

We want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Designers have until **July 2** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of July.

We can't wait to see what amazing designs you come up with. 

Good luck to all participants! 🌻

Last Bumpkin Competition: https://github.com/sunflower-land/sunflower-land/discussions/2552

**Comment by digital-farms** [comment]
# Introducing a collection of stylish glasses for your bumpkin character!
_1. Lovely Eyes
2. Steve Style
3. Beetroot Party
4. Pink Look
5. Pixelz
6. Space Glasses_

![Glasses](https://user-images.githubusercontent.com/105886943/232482964-a7a91710-ce76-47c2-9549-87215b18b2b4.png)

# Assets:
↘️ Lovely Eyes
![Lovely Eyes](https://github.com/sunflower-land/sunflower-land/assets/105886943/8651c57c-355d-4ea5-b360-991d0e6f6e9e)

↘️ Steve Style 
![Steve Style](https://github.com/sunflower-land/sunflower-land/assets/105886943/10686152-d662-430f-ace3-1901cfaaec2e)

↘️ Beetroot Party
![Beetroot Party](https://github.com/sunflower-land/sunflower-land/assets/105886943/2f5a6204-7851-49b5-a3a3-af7f2ea5ae4b)

↘️ Pink Look
![Pink Look](https://github.com/sunflower-land/sunflower-land/assets/105886943/f21776d8-37af-4ad6-9eb7-c6f7357baf44)

↘️ Pixelz
![Pixelz](https://github.com/sunflower-land/sunflower-land/assets/105886943/51d332bb-c3e7-418c-b21b-549e405ccd82)

↘️ Space Glasses
![Space Glasses](https://github.com/sunflower-land/sunflower-land/assets/105886943/5416a34c-b7c5-446e-abf2-d73dc876db74)

`land id: 21637`
`discord: KOTOB#4391`

**Comment by LTNGx39** [comment]
### <h1> Old diving suit
_**A set arrived from the depths** (literally)_

`Farm ID: 161769`
`Discord: ncnlx39`

**Preview:**
![Preview 3](https://github.com/sunflower-land/sunflower-land/assets/131133076/069b51d1-d782-48fd-b50e-3b054450b55b)

| Tool | Hat | Shirt | Pants | Shoes |
| - | - | - | - | - |
| ![Tool](https://github.com/sunflower-land/sunflower-land/assets/131133076/9779af90-98bb-4de8-bf15-e27b823dbac2) | ![Hat](https://github.com/sunflower-land/sunflower-land/assets/131133076/836bf978-8e62-4c0c-8b5a-2927bae6d0d7) | ![Shirt](https://github.com/sunflower-land/sunflower-land/assets/131133076/5214982b-ce9e-4acd-826d-e9779410723f) | ![Pants](https://github.com/sunflower-land/sunflower-land/assets/131133076/a0fb27c3-fea4-4ddd-9cb6-aa79bbfe35c3) | ![Shoes](https://github.com/sunflower-land/sunflower-land/assets/131133076/0c3fe655-4de8-41c7-a472-f89adafd166c) |

**Animation:**

![animacion](https://github.com/sunflower-land/sunflower-land/assets/131133076/8cdc3a04-c75a-40de-a89b-a28b4072f08f)

| Sprite |
| - |
| ![css_sprites](https://github.com/sunflower-land/sunflower-land/assets/131133076/832e665a-c571-4907-860c-63fa4448cf99) |

**Examples and combinations:**

| Tool + Pants + Shoes | Tool + Shirt + Shoes | Tool + Hat |
| - | - | - |
| ![Example 1](https://github.com/sunflower-land/sunflower-land/assets/131133076/5ea8c7ff-feb1-4db3-bf5a-580e79001f09) | ![Example 2](https://github.com/sunflower-land/sunflower-land/assets/131133076/da3f05f2-8840-4be0-b6d9-c5771425cc7b) | ![Example 3](https://github.com/sunflower-land/sunflower-land/assets/131133076/f2b45ae2-b3a2-4227-8179-bd19b5060e93) |

**Comment by mmaxxpeinn** [comment]
**Pilot Uniform**

![pilot_bumpkin](https://github.com/sunflower-land/sunflower-land/assets/120295844/7380e94c-d0a7-4fb4-b502-baf32908a25b)

|hat|masked hat|uniform|pants|shoes|
|---|---|---|---|---|
|![pilot_hat_bumpkin](https://github.com/sunflower-land/sunflower-land/assets/120295844/07fc63cc-901c-466d-b8d1-95e3b6260aeb)|![pilot_hat_bumpkin_masked](https://github.com/sunflower-land/sunflower-land/assets/120295844/db4b73af-a6dc-442a-b88e-f0028b4d74f2)|![pilot_shirt_bumpkin](https://github.com/sunflower-land/sunflower-land/assets/120295844/c7104b09-4cac-4b9b-8a37-67d807ca322d)|![pilot_pants_bumpkin](https://github.com/sunflower-land/sunflower-land/assets/120295844/8c847d0a-d5c1-4dea-af30-ea9235152238)|![pilot_shoes_bumpkin](https://github.com/sunflower-land/sunflower-land/assets/120295844/d1f5024b-f0ec-47e2-9a24-49f3171c61ea)|

*npc preview*
![pilot_npc_land](https://github.com/sunflower-land/sunflower-land/assets/120295844/eb5de77f-3379-467c-95bb-3b481bfe1a96)

|hat|![pilot_hat_npc_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/0c0e929d-33f6-4728-b47e-41db37fb04f6)|
|---|---|
|uniform|![pilot_uniform_npc_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/b03bd65c-1bdb-4869-a9bd-63cbd9038179)|
|pants|![pilot_pants_npc_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/e6a40e74-f5c0-4769-8732-a8f6007ebac7)|

**Female Viking**

![female_viking](https://github.com/sunflower-land/sunflower-land/assets/120295844/d9ffe4cc-de8f-43b9-85bc-cc93a0ea0c5d)

|hat|masked hat|suit|skirt|shoes|shield|
|---|---|---|---|---|---|
|![viking_hat](https://github.com/sunflower-land/sunflower-land/assets/120295844/70d74490-1be4-4faa-b37d-dc172776bf7f)|![masked_viking_hat](https://github.com/sunflower-land/sunflower-land/assets/120295844/5d4da3e0-f78e-4fa5-88a6-986a1dfc6226)|![viking_suit](https://github.com/sunflower-land/sunflower-land/assets/120295844/9c47f639-1973-4f37-89be-db2f67e1a45b)|![viking_skirt](https://github.com/sunflower-land/sunflower-land/assets/120295844/17f9ee53-f3ca-48f6-949a-aa7910b21f5b)|![viking_shoes](https://github.com/sunflower-land/sunflower-land/assets/120295844/83706d87-6f35-4ec7-85e2-78b4d2ccf069)|![viking_shield](https://github.com/sunflower-land/sunflower-land/assets/120295844/46b58179-b243-41d5-8dc2-4bb519fc8b54)|

*npc preview*
![viking_npc_land](https://github.com/sunflower-land/sunflower-land/assets/120295844/d33e44d0-1d81-47c2-96f4-e0609a3a274a)

|hat|![viking_hat_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/95621616-f02a-4b61-982d-fbfc26540bf9)|
|---|---|
|suit|![viking_suit_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/292442dd-5e90-4811-aaa7-180364198434)|
|skirt|![viking_skirt_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/35670e5e-c9f9-4958-8def-37a2f554e609)|

Farm ID: `82122`
Discord: `mmaxxpeinn`

**Comment by peperol** [comment]
I'm sorry, did someone say....beach party? ;)
![ on the beach](https://github.com/sunflower-land/sunflower-land/assets/114354077/aa8e0af8-7682-4e17-956e-8079fa2b8291)

![beach hat](https://github.com/sunflower-land/sunflower-land/assets/114354077/9114624a-e050-4d4b-8076-4a1ad3137d58)
cutie strawberry inflatable
![cutie strawberry inflatable](https://github.com/sunflower-land/sunflower-land/assets/114354077/d0264262-2213-4370-8abf-5449db9837d6)
beach top
![beach top](https://github.com/sunflower-land/sunflower-land/assets/114354077/b20a581e-71aa-4b94-9b79-6cdeaa63ce46)
beach pareo
![pareo](https://github.com/sunflower-land/sunflower-land/assets/114354077/8085d6eb-3b78-432e-a5f6-4de86b2ceaad)
beach hair
![beach hair](https://github.com/sunflower-land/sunflower-land/assets/114354077/fad7234a-aca7-4d66-a0d6-247887ac9527)

beach goblin girl
![goblin beach](https://github.com/sunflower-land/sunflower-land/assets/114354077/a30a6ecf-15f9-4908-90bf-f0d0074d8905)

 
![beach girl](https://github.com/sunflower-land/sunflower-land/assets/114354077/04880b5e-b848-41d9-ba76-f49ab94984f9)

beach girl hat
![prova hat ](https://github.com/sunflower-land/sunflower-land/assets/114354077/474022f2-5cd1-438a-8f7b-4d8474b2d429)

in land
![beach outfit in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/966f6d0d-ce6c-4a5b-bac6-2933b58f3bf9)

![beach hat in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/af4d4510-7473-4223-ac44-be9430f5fd40)

Rockabilly style

![rockabilly girl](https://github.com/sunflower-land/sunflower-land/assets/114354077/346f5756-0667-4bee-8e24-3bfe1d6b85bf)
high waist shorts Rockabilly Style
![high waist shorts Rockabilly Style](https://github.com/sunflower-land/sunflower-land/assets/114354077/70173bfd-0cc7-4ce1-a114-01ea29621aa3)
rockabilly top
![top Rockabilly Style](https://github.com/sunflower-land/sunflower-land/assets/114354077/4d8d41ba-478e-486d-bfa6-0ec926cdd89e)
rockabilly skin
![skin Rockabilly Style](https://github.com/sunflower-land/sunflower-land/assets/114354077/f06ece45-1168-4227-9294-859b37bab823)

rockabilly hair
![rockabilly hair style](https://github.com/sunflower-land/sunflower-land/assets/114354077/556fa07b-0012-41fd-a353-0c87a0d11a54)
rockabylly tattoo
![Tattoo](https://github.com/sunflower-land/sunflower-land/assets/114354077/189d53bb-edbb-4ca5-b99e-4be0eaaf402f)
drink
![drink](https://github.com/sunflower-land/sunflower-land/assets/114354077/1275831e-334e-4e9d-bd89-36950dc21e87)

![rockabilly girl](https://github.com/sunflower-land/sunflower-land/assets/114354077/29d8f2c8-5974-4812-a7ab-2ba316d35d58)
in land
![RB on land](https://github.com/sunflower-land/sunflower-land/assets/114354077/64463801-4f4c-49c7-8a2a-e303c3b4022a)

Ice cream gnammy
![screen](https://github.com/sunflower-land/sunflower-land/assets/114354077/4dc520bb-c44d-435a-9808-1c103e4a4f22)

beckground
![IC shop ](https://github.com/sunflower-land/sunflower-land/assets/114354077/7c3903bf-083d-4b1f-a81a-eb912a0cd19d)


shirt
![camicia](https://github.com/sunflower-land/sunflower-land/assets/114354077/0f622423-3363-4325-8f9d-391addae3ace)
chocolate skirt
![chocolate skirt](https://github.com/sunflower-land/sunflower-land/assets/114354077/4f1c0b3e-26c0-4986-8277-1d631cdc425c)
cono hat
![cappello cono](https://github.com/sunflower-land/sunflower-land/assets/114354077/4b70cd0a-d501-442e-a81f-978f68c0da4a)
ice cream hair
![ice cream hair](https://github.com/sunflower-land/sunflower-land/assets/114354077/1b888896-63a4-4cab-8dca-374c960380b7)
shoes
![scarpe corda](https://github.com/sunflower-land/sunflower-land/assets/114354077/18c52133-6633-4134-ad42-196be0dbeaa5)
spiteful cherry 
![cherry](https://github.com/sunflower-land/sunflower-land/assets/114354077/2eef8a24-e47b-470c-9b18-a8c3f4008e12)

![ice cream girl](https://github.com/sunflower-land/sunflower-land/assets/114354077/c12da204-f5e1-4d6e-864b-f121d9cd06f7)
in land
![ice creamin land](https://github.com/sunflower-land/sunflower-land/assets/114354077/adb2e51c-2693-48af-873e-fee0dedde1f2)


Farm ID:145849
Discord: peperol
